web/src/engine/websites/decorators/FoolSlide.ts

```diff
@@ -38,7 +46,17 @@ export async function FetchMangaCSS(this: MangaScraper, provider: MangaPlugin, u
  * @param query - A CSS query to locate the element from which the manga title shall be extracted
  */
 export function MangaCSS(pattern: RegExp, query: string = queryMangaTitle) {
```

Comment:
Have all other FoolSlide website using `import * as FoolSlide` (e.g., AnataNoMotokare, DeathTollScans, ...) been tested for regression to still work after this change?

Reply:
Of course boss. Both SFW and NSFW works. That say, some website now have expired certificate and couldnt be tested 

anatanomotokare : no more reader
patyscans: certificate expired
PCnet : certificate expired
silentskyscans : dead